### DISCUSSION TITLE
General Discussions!

### DISCUSSION THREAD

**Discussion by smilewithkhushi** [smilewithkhushi/BasicNative#37] [General]
Want to share your experience in contributing to this repository, any issues your faced or something else? Feel free to drop your thoughts here!

**Comment by smilewithkhushi** [comment]
Hey y'all! I'll be merging the PR's in 2-3 days. I've currently my end sems ongoing and there's a few conflicts which need to be resolved.
I hope you'll stay patient, and dw all your PRs will be merged :)))

**Reply by saikrishna823** [reply]
Please check my Pull Request.I hope you will consider my application,since I submitted first.

**Comment by saikrishna823** [comment]
@smilewithkhushi ,Could you please rename the label?

**Comment by KAmaL-senpai** [comment]
Can i get the link of the BasicNative web page i am not unable to find it?

**Comment by saikrishna823** [comment]
It is not a website,an app.Just clone the repo and
 run npm i - to install all the libraries and dependencies 
download expo app from appstore/playstore 
run npx expo start

**Reply by KAmaL-senpai** [reply]
okay Thanks for the info.

**Comment by the-matrixneo** [comment]
i just raised the issue regarding TODO list  its showing Greeting : All Jobs Failed in 7 Seconds . Can you please help me.
